Readme.md

```diff
@@ -1,10 +1,18 @@
+<div id="header" align="center">
+
+[![GitHub issues](https://img.shields.io/github/issues-raw/eclipse/che-che4z-lsp-for-hlasm?style=flat-square)](https://github.com/eclipse/che-che4z-explorer-for-endevor/issues)
```

Comment:
the link in here needs to point to explorer for endevor repo, otherwise the button will show number of issues from HLASM repo

Reply:
@DStatWriter